go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/256dpi/gomqtt v0.14.2
 	github.com/bettercap/gatt v0.0.0-20191018133023-569d3d9372bb
 	github.com/davecgh/go-spew v1.1.1
+	github.com/eclipse/paho.MQTT.golang v1.2.0
```

Comment:
Why revert MQTT lib back to v1.2.0? We have already bumped `github.com/eclipse/paho.mqtt.golang` in https://github.com/cnrancher/octopus/commit/4bfb6e1e1a1309bcf49ad952e0fffac307757b5c.

Reply:
updated